hermes/src/main/java/in/raveesh/hermes/Util.java

```diff
@@ -0,0 +1,16 @@
+package in.raveesh.hermes;
+
+import android.annotation.SuppressLint;
+import android.util.Log;
+
+/**
+ * Created by sahil on 15/5/15.
+ */
+public class Util {
+    public static final String LOG_TAG = "in.raveesh.hermesexample";
+
+    @SuppressLint("LongLogTag")
+    public static void log(String message) {
```

Comment:
I'll be having a look at the entire pull request later at night. I would, however, be leaving out Utils and logging the way you've done it, since my preference is to leave as many things to be on platform standards vs personal programming styles for open source projects. For eg, Utils is a common class name and might show up as an option when a bunch of users try importing their own classes. Easier to just avoid the entire thing.